date: 12/02/2021 20:34:56, exception type:<class 'models.UnableToWorkException'>, Message: I'm not hired yet, lol., 
traceback: <traceback object at 0x000002368B8F8700>
date: 12/02/2021 20:36:35, exception type:<class 'ValueError'>, Message: , 
traceback: <traceback object at 0x0000019D94B58940>
date: 12/02/2021 20:37:33, exception type:<class 'ValueError'>, Message: , 
traceback: <traceback object at 0x000001BCFD2ED500>
date: 12/02/2021 20:37:39, exception type:<class 'ValueError'>, Message: , 
traceback: <traceback object at 0x00000247758FD140>
